nuclia/sdk/extract_strategy.py

```diff
@@ -0,0 +1,104 @@
+from nuclia.data import get_auth, get_async_auth
+from nuclia.decorators import kb, pretty
+from nuclia.exceptions import InvalidPayload
+from nuclia.lib.kb import NucliaDBClient, AsyncNucliaDBClient
+from nuclia.sdk.auth import NucliaAuth, AsyncNucliaAuth
+from nuclia_models.config.proto import ExtractConfig
+
+
+class NucliaExtractStrategy:
+    @property
+    def _auth(self) -> NucliaAuth:
+        auth = get_auth()
+        return auth
+
+    @kb
+    @pretty
+    def list(self, *args, **kwargs) -> ExtractConfig:
+        """
+        List extract strategies
+        """
+        ndb: NucliaDBClient = kwargs["ndb"]
+        response = ndb.list_extract_strategies()
+        return response.json()
+
+    @kb
+    def add(
+        self,
+        *args,
+        config: ExtractConfig,
+        **kwargs,
+    ):
+        """
+        Add extract strategy
+
+        :param config: strategy configuration
+        """
+        if isinstance(config, dict):
+            config = ExtractConfig.model_validate(config)
+
+        ndb: NucliaDBClient = kwargs["ndb"]
+        response = ndb.add_extract_strategy(config=config)
+        return response.json()
```

Comment:
you don't need to return anything on the add method no? Otherwise, add the corresponding return type on the signature